src/platform-implementation-js/views/conversations/thread-view.ts

```diff
@@ -143,6 +143,10 @@ class ThreadView extends (EventEmitter as new () => TypedEventEmitter<ThreadView
   addLabel(): SimpleElementView {
     return get(memberMap, this).threadViewImplementation.addLabel();
   }
+
+  addSubjectButton(buttonDescriptor: Record<string, any>) {
```

Comment:
Could we use https://github.com/InboxSDK/InboxSDK/blob/4edf7ccd5e69417933291d69006d111f7bfc4bf7/src/inboxsdk.d.ts#L446 here?

```suggestion
  addSubjectButton(buttonDescriptor: Descriptor<ButtonDescriptor>) {
```

Reply:
If we don't necessarily want an `Observable` as part of the API, an alternative would be 

```suggestion
  addSubjectButton(buttonDescriptor: ButtonDescriptor) {
```